go/node/node.go

```diff
@@ -0,0 +1,5 @@
+package node
+
+type Node interface {
```

Comment:
What uses this interface? Very hard to search and check, 226 matches in the browser search haha